docs/common/craft-parts/craft-parts.wordlist.txt

```diff
@@ -65,6 +65,7 @@ DotPluginEnvironmentValidator
 Dotnet
 DotnetPlugin
 DotnetPluginProperties
+DotnetV
```

Comment:
There are several words added here that are:

- In source code
- Marked up as code in the docs (\`\`example\`\`)

The spell check shouldn't be picking these up, so we can safely remove them from this file.

Reply:
Ah, I think it's actually an issue with the way our spellcheck is set up. Not a problem. No further action needed. :)